package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-js",
-  "version": "0.0.2",
+  "version": "0.0.6",
```

Comment:
Since this is all going in a single PR, can't the version be bumped from 0.0.2 to 0.0.3 rather than 0.0.6?

Reply:
The issue is that for testing the NPM packages, I published different versions already. 0.0.6 is the current one (see the commits)

Reply:
Normally we won't do this after is merged.

- short term: we'll do a version in master branch (develop for dex-react or dex-telegram), and publish that
- later: we'll integrate with travis, so it's travis the one publishing on every new tag